install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -39,3 +39,6 @@ ALTER TABLE `PREFIX_log`
   ADD `id_lang` INT(10) NULL DEFAULT NULL after `id_shop_group`,
   ADD `in_all_shops` TINYINT(1) unsigned NOT NULL DEFAULT '0'
 ;
+ALTER TABLE `PREFIX_stock_available` MODIFY COLUMN `location` VARCHAR(255) DEFAULT NULL;
```

Comment:
You need to update the `db_structure.sql` file as well It might be the reason why all the tests are failing